src/resources/views/corporation/ledger/reprocessingfeebymonth.blade.php

```diff
@@ -0,0 +1,70 @@
+@extends('web::corporation.ledger.layouts.view', ['sub_viewname' => 'bountyprizesbymonth'])
+
+@section('title', trans_choice('web::seat.corporation', 1) . ' ' . trans_choice('web::seat.bountyprizesbymonth', 2))
+@section('page_header', trans_choice('web::seat.corporation', 1) . ' ' . trans_choice('web::seat.bountyprizesbymonth', 2))
+
+@section('ledger_content')
+
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Available Ledgers</h3>
+    </div>
+    <div class="panel-body">
+
+      @foreach ($bountyprizes->chunk(3) as $chunk)
+        <div class="row">
+
+          @foreach ($chunk as $prize)
+            <div class="col-xs-4">
+              <span class="text-bold">
+                <a href="{{ route('corporation.view.ledger.bountyprizesbymonth', ['corporation_id' => $corporation_id, 'year' => $prize->year, 'month' => $prize->month]) }}">
+                  {{ date("M Y", strtotime($prize->year."-".$prize->month."-01")) }}
+                </a>
+              </span>
+            </div>
+          @endforeach
+
+        </div>
+      @endforeach
+    </div>
+  </div>
+
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">{{ trans_choice('web::seat.bountyprizesbymonth', 2) }}
+        - {{ date("M Y", strtotime($year."-".$month."-01")) }}</h3>
+    </div>
+
+    <div class="panel-body">
+      <div>
+        <table class="table datatable table-condensed table-hover table-responsive">
+          <thead>
+          <tr>
+            <th>{{ trans_choice('web::seat.name', 1) }}</th>
+            <th>{{ trans_choice('web::seat.bountyprizetotal', 1) }}</th>
+          </tr>
+          </thead>
+          <tbody>
+
+          @foreach ($bountyprizedates as $bpbm)
```

Comment:
`bountyprizedates` does not exist here. It have to be `reprocessingfeedates` according to your controller.